src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -239,7 +276,7 @@ public function returnImage($id, $formatName)
      *
      * @throws ImageProxyInvalidImageFormat
      */
-    protected function getFormat($format)
+    private function getFormat($format)
```

Comment:
Any reason you have done this? That's quite a big BC break... Although I am quite sure nobody has overridden this class I don't see why we should take this risk.